applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py

```diff
@@ -21,9 +21,12 @@ def CreateSolver(main_model_part, custom_settings):
         elif (solver_type == "FractionalStep"):
             solver_module_name = "navier_stokes_solver_fractionalstep"
 
-        elif ((solver_type == "Embedded") or (solver_type == "EmbeddedAusas") or (solver_type == "EmbeddedDevelopment")):
+        elif ((solver_type == "Embedded") or (solver_type == "EmbeddedDevelopment")):
             solver_module_name = "navier_stokes_embedded_solver"
 
+        elif (solver_type == "EmbeddedAusas"):
```

Comment:
I recommend to use style-guide consistent names if you introduce a new solver. (=> `embedded_ausas`)
Or as a temp solution: `elif (solver_type == "EmbeddedAusas" or solver_type = embedded_ausas):`

Reply:
We thought to change all the solver names together with the interface.